frontend/src/pages/devices/List.tsx

```diff
@@ -216,28 +215,16 @@ export const List = () => {
                         const minutesBetween = (wc - sa) / 1000 / 60;
                         if (expectedResponseInMinutes < minutesBetween) {
                             const distance = formatDistance(sa, wc);
-                            warnings.push(<Badge variant="danger">Slow to Respond (took { distance } to add code { lc.code })</Badge>);
+                            warnings.push(<p className="font-weight-bold">Slow to Respond (took { distance } to add code { lc.code })</p>);
```

Comment:
Do we need the text bolded? Please share a screen shot with it bolded and plain.

Reply:
![image](https://user-images.githubusercontent.com/54916525/187058583-06aa5dbc-9a41-4278-ae65-cb730b88f50d.png)
![image](https://user-images.githubusercontent.com/54916525/187058633-a84c33de-54c4-480c-a47b-b077099895f8.png)
Think it may look better without bold

Reply:
Let's roll with out the bold (sharing a screenshot of the whole page would also help me see how it fits into everything else 😉).

Reply:
Realized that after I sent it

Reply:
Is this the only function that populates the `Status` column with warnings?

Reply:
It's all rendered by renderEntityStatus() which is multiple functions, right?

Reply:
Yeah, `renderEntityStatus` calls multiple functions that create warnings. You might want to check them all for `Badge`s that will be colorful.